### DISCUSSION TITLE
Question before I prune a node

### DISCUSSION THREAD

**Discussion by daank-c** [dogecoin/dogecoin#3449] [Q&A]
I am about to prune a node, and have a question...

I'm wondering if it would be sensible to take the hash of a block that I currently have validated, and use that as an `assumevalid=<hex>` in my config file, to speed up the re-download.

I figure I'm not trusting anyone to do this, as I've already validated the block on my own. I can use the hash of a block from a week ago or something, so I don't need to worry about reorgs.

**Comment by patricklodder** [comment]
If you already have the entire blockchain, it will not redownload it.

**Reply by daank-c** [reply]
My mistake, I misread! Thanks, for the clarification.
